framework/source/class/qx/test/Interface.js

```diff
@@ -173,7 +171,7 @@ qx.Class.define("qx.test.Interface",
       if (this.isDebugOn())
       {
         this.assertException(function() {
-          var i = new qx.test.i.ICar();
+          qx.test.i.ICar();
```

Comment:
Is this really equivalent? Does simply writing ```qx.text.i.ICar()``` create a new instance of that class? I'm asking because I really don't know that.